docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  selenium:
+    image: selenium/standalone-chrome
```

Comment:
Is there a reason we updated the selenium image used from edge? ref https://github.com/Azure/ARO-RP/blob/d64782c410f1155a79047849b5d1fb8900242c28/hack/e2e/run-rp-and-e2e.sh#L108

Reply:
Switched Selenium image to standalone-chrome for consistency across configuration

Reply:
Where is standalone-chrome used today?

Reply:
good point. Reverted to selenium/standalone-edge for consistency. Thanks for catching that!